java-runtime-builder-app/src/test/resources/testWorkspaces/customBuildWorkspace/app.yaml

```diff
@@ -0,0 +1,4 @@
+env: flex
+runtime: java
+runtime_config:
+  build_script: "bazel build //:all"
```

Comment:
How does this work? Who calls this script? Wouldn't bazel need to be installed in the builder container?